apps/libs/digest.h

```diff
@@ -0,0 +1,34 @@
+#ifndef TOCK_DIGEST_H
+#define TOCK_DIGEST_H
+
+#include <stdlib.h>
+
+// TODO These need to be standardized
+#define HOTEL_DRIVER_DIGEST 2
```

Comment:
Yeah, that's pretty much what I had in mind. I'd only move the platform drivers to something like 0x10000 instead, because negative number could be re-purposed to mean something else.